config/settings.yml

```diff
@@ -1254,6 +1254,7 @@
       :ems_metrics_collector_worker:
         :defaults:
           :count: 2
+          :memory_threshold: 400.megabytes
```

Comment:
@h-kataria Sorry, This number is incorrect.  From master before this change:

```ruby
irb(main):004:0> MiqEmsMetricsCollectorWorker.worker_settings[:memory_threshold]
=> 524288000
```

Nearly all code outside of this page honor the inheritance of worker_settings.  In this case, this worker was inheriting the value from it's parent class queue_worker_base, which has changed in the many months since this BZ was opened to 500.megabytes, see line 1251 above.

Note, there are several ways to see if the value for a specific worker is inherited from the parent. 

One way is to compare the superclass's value to yours:

```ruby
irb(main):005:0> MiqEmsMetricsCollectorWorker.worker_settings[:memory_threshold] == MiqEmsMetricsCollectorWorker.superclass.worker_settings[:memory_threshold]
=> true
```

On this screen, I think you only want to know if a specific value is "changed" and/or "inherited".  You don't really care WHERE up the inheritance tree the worker got the value, only if it's inherited or customized for just itself.

If we aren't going to honor the inheritance of worker settings in this page, this problem will happen again when we change some other workers to use a default from a parent class.  Note, I'm fine with doing this change for now to fix this bug if we come back and either drop inheritance of worker settings or add support for it in this screen later.

If we need backend methods to help implement this correctly in the UI, please let me know.

Or, if we think this is too confusing to users, let's just remove this feature entirely.  I'm pretty surprised since this has been this way without issue for many years.